libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -33,8 +36,19 @@
     textToSpeech = new TextToSpeech(context, new TextToSpeech.OnInitListener() {
       @Override
       public void onInit(int status) {
-        if (status != TextToSpeech.ERROR) {
-          textToSpeech.setLanguage(new Locale(language));
+        if (status == TextToSpeech.ERROR || language == null) {
+          Timber.w("There was an error initializing native TTS");
+          textToSpeech = null;
```

Comment:
Is there a reason to set TTS to `null` in the error or unavailable `Locale` scenarios?  It looks like `languageSupported` will be `false`, so the `null` `textToSpeech` won't be touched, but I was curious if there was a reason behind this.